server/shared/datasheet.js

```diff
@@ -1,9 +1,7 @@
-'use strict';
-
-const { camelCase } = require('change-case');
-const intoStream = require('into-stream');
-const { Workbook } = require('exceljs');
-const Worksheet = require('exceljs/dist/es5/doc/worksheet');
+import { camelCase } from 'change-case';
```

Comment:
@malizeus I think `datasheet.js` has been improved a lot in `boutique`. Maybe you could check it and implement these improvements in this stater. 👀